src/lib/process/scheduler.ts

```diff
@@ -92,18 +91,19 @@ export class ProcessScheduler {
 		return this.processes.find((p) => p.name === name);
 	}
 
-	private start(): void {
-		if (!this.isStarted) {
-			this.isStarted = true;
-			this.connection = RunService.Heartbeat.Connect(() => this.onHeartbeat());
-		}
+	public start(): void {
+		if (this.connection) return; // TODO: log that they're doing something stupid.
+		this.connection = RunService.Heartbeat.Connect(() => this.onHeartbeat());
 	}
 
-	private stop(): void {
-		if (this.isStarted) {
-			this.isStarted = false;
-			this.connection?.Disconnect();
-		}
+	/**
+	 * @hidden
+	 * Remove all references and disconnect all connections
```

Comment:
The main flaw with this is that the running code would still have to get rid of this reference itself, so if they declared it as `const process = Tina.process(...)` then they're still going to be holding that reference. Ah well, we'll keep this for the future. #18